Logger/FileLogger.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.IO;
+
+namespace Logger;
+internal class FileLogger : BaseLogger
+{
+    private readonly StreamWriter LogFile;
+    public FileLogger(string logPath) {
+        LogFile = new StreamWriter(logPath, true);
+    }
```

Comment:
```suggestion
    private readonly string _FilePath;
    public FileLogger(string filePath) {
        _FilePath= filePath;
    }
```

I would not recommend setting up a StreamWriter here. I'd recommend only using your stream writer in your log method and then disposing of it. Doing it this way can create concurrency issues.